draft-ietf-quic-http.md

```diff
@@ -1035,11 +1035,9 @@ integer at the start of the stream. The format and structure of data that
 follows this integer is determined by the stream type.
 
 ~~~~~~~~~~ drawing
- 0                   1                   2                   3
- 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                        Stream Type (i)                      ...
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+Unidirectional Stream Header {
+  Type (i),
```

Comment:
```suggestion
  Stream Type (i),
```
To avoid changing the name or be confused with the 'Type' used in all the frames.